app/controllers/dev_controller.rb

```diff
@@ -32,6 +32,8 @@ def reset
       )
     end
 
+    Flipper.enable :new_consents
```

Comment:
Will this be removed before merging?

Reply:
We need to keep it in because tests will fail otherwise, since they've been migrated to the new controller and certain edge cases have been updated. It will be removed in the follow-up PR that removes the feature flag completely.